ifacemonitor/iface_monitor.go

```diff
@@ -44,6 +44,9 @@ type InterfaceStateCallback func(ifaceName string, ifaceState State)
 type AddrStateCallback func(ifaceName string, addrs set.Set)
 
 type Config struct {
+	// List of interface names that should be excluded from being monitored on events.
+	// Dataplane recieves link add/delete events for an excluded interface as a normal interface.
+	// Dataplane ignores ip address update and link up/down events for an excluded interface.
```

Comment:
I think this is still difficult to understand.  The reader would have to know that "link add/delete events" means AddrStateCallback with no address information, and that "link up/down events" means InterfaceStateCallback.

The best thing, I think, would be if we can eliminate the one remaining AddrStateCallback.  Then we can just say that there are no callbacks at all for an excluded interface.